src/main/java/com/playtika/homework2/task2/ManipulationsWithListOfStudent.java

```diff
@@ -0,0 +1,88 @@
+        String[] firstName = {"Андрей", "Павел", "Евгений", "Анастасия", "Юрий", "Ольга", "Александр", "Артем", "Денис", "Алиса"};
+        String[] lastName = {"Попков", "Жуковский", "Невмержицкий", "Бартош", "Хизюк", "Соболева", "Кондратович", "Петров", "Петров", "Иванова"};
+        int[] age = {21, 19, 22, 18, 17, 22, 19, 20, 20, 23};
+        List<Student> students = new ArrayList<>();
+
+        int countLastNames = lastName.length;
+        System.out.println("Список студентов:\n");
+        for (int i = 0; i < countLastNames; i++) {
+            students.add(new Student(id[i], firstName[i], lastName[i], age[i]));
+            System.out.println(students.get(i));
+        }
+
+        //    Метод для сортировки по возрасту
+        students.sort(new Comparator<Student>() {
+            @Override
+            public int compare(Student o1, Student o2) {
+                if (o1.getAge() == o2.getAge()) {
+                    return 0;
+                }
+                return o1.getAge() < o2.getAge() ? -1 : 1;
+            }
+        });
+
+        System.out.println("\n\nСортировка по возрасту:\n");
+        for (Student student : students) {
+            System.out.println(student);
+        }
+
+        //Вывод студента по первой букве фамилии
+        System.out.println("\n\nВведите первую букву фамилии\n");
+        Scanner sc = new Scanner(System.in);
+        String letter = sc.nextLine();
+        int flag = 0;
+        for (Student student : students) {
+            if (student.getLastName().startsWith(letter)) {
+                System.out.println(student);
+                flag++;
+            }
+        }
+        if (flag == 0) {
+            System.out.println("Не найдено фамилии начинающейся с ведденой буквы");
+        }
+
+        // Средний возраст студентов
+        int sumAge = 0;
+        int count = 0;
+        for (int i = 0; i < students.size(); i++) {
+            sumAge += age[i];
+            count++;
+        }
+        int averageAge = sumAge / count;
+        System.out.println("\n\nСредний возраст студентов: \n" + averageAge);
+
+        // Преобразование в Map
+        System.out.println("\n\nСписок студентов из Map: \n");
+        Map<Integer, Student> mapStudent = new HashMap<>();
+        for (int i = 0; i < id.length; i++) {
+            mapStudent.put(id[i], students.get(i));
+        }
+        for (Map.Entry entry : mapStudent.entrySet()) {
+            Student mapSt = (Student) entry.getValue();
+            System.out.println(entry.getKey() + " " + mapSt.getFirstName() + " " + mapSt.getLastName());
+        }
+
+        // Вывод студентов у которых id больше заданного значени
+        System.out.println("\n\nВывод студентов у которых Id больше введенного значения: \n");
+        System.out.println("Введите Id: ");
+        Scanner scan = new Scanner(System.in);
+        int idForFound = scan.nextInt();
+        for (Map.Entry entry : mapStudent.entrySet()) {
```

Comment:
`for (Map.Entry<Integer, Student> entry : mapStudent.entrySet()) {`
remove casting on the next line